caddy/Caddyfile

```diff
@@ -5,6 +5,7 @@
     reverse_proxy /caluma-media* http://minio:9000
     reverse_proxy /minio/* http://minio:9000
     reverse_proxy /* http://ember:80
+    reverse_proxy /static/graphene_django/* http://caluma:8000
```

Comment:
I don't like graphiql to be exposed in prod. I'd prefer to have this only in dev. We can use env vars in the caddy file, maybe that would be the way to go. If you don't know how to achieve that, you can also just remove this change from the PR.

Reply:
Something like this?
Caddyfile
```
{$GRAPIQL_PROXY}
```
.caddy
```
GRAPIQL_PROXY="reverse_proxy /static/graphene_django/* http://caluma:8000"
```

Reply:
If it works, yes, awesome!

Two things:

1. Typo in env var (should be `GRAPHIQL_PROXY`)
2. Don't put the env var into `.caddy`, but set it in the compose override.

Reply:
Done